docs/data_encryption.md

```diff
@@ -0,0 +1,253 @@
+Encrypting your data
```

Comment:
@bacongobbler Take a look at this documentation and let me know if you think it is adequate.